### PR TITLE
change the wording around the signature description

### PR DESCRIPTION
**Pull request by tippenein**
> Traits are defined with a name, and a list functions, defined with a name, a list of argument types, and return type.

I had to read this line a couple times to understand what it was trying to say. It might actually be better to delete it entirely since it's trying to describe the `signature` which is directly above it. Not sure.. Thought I'd open this at least to a discussion. The opening sentence feels a bit redundant too. "`define-trait` is used to define a new trait"

### CODE REVIEW

**Comment by tippenein** [comment]
context from discord: https://discord.com/channels/621759717756370964/623217767356694547/1183566565199843378

File: clarity/src/vm/docs/mod.rs [hunk 1]
```diff
@@ -2001,7 +2001,7 @@ const DEFINE_TRAIT_API: DefineAPI = DefineAPI {
 can implement a given trait and then have their contract identifier being passed as a function argument in order to be called
 dynamically with `contract-call?`.
 
-Traits are defined with a name, and a list functions, defined with a name, a list of argument types, and return type.
+Traits are defined with a name, a list of functions, and a return type. The list of functions here is defined with a name and the list of argument types.
```
Comment (obycode) [inline clarity/src/vm/docs/mod.rs:2004]: Hi @tippenein. Thanks for trying to make this more clear! The return type should go with the functions. Would something like this be more clear?

```
Traits are defined with a name, and a list functions, where each function is defined with a name, a list of argument types, and a return type.
```

Comment (tippenein) [reply clarity/src/vm/docs/mod.rs:2004]: Yeah that's better imo

File: clarity/src/vm/docs/mod.rs [hunk 2]
```diff
@@ -2001,7 +2001,7 @@ const DEFINE_TRAIT_API: DefineAPI = DefineAPI {
 can implement a given trait and then have their contract identifier being passed as a function argument in order to be called
 dynamically with `contract-call?`.
 
-Traits are defined with a name, and a list functions, defined with a name, a list of argument types, and return type.
+Traits are defined with a name, and a list functions, where each function is defined with a name, a list of argument types, and a return type.
```
Comment (obycode) [inline clarity/src/vm/docs/mod.rs:2004]: Thanks! While you're here, I just realized we're missing an "of":
"... and a list **of** functions,..."

**Review by obycode — DISMISSED** [review]
🎉 Thanks for that back and forth!

**Comment by obycode** [comment]
Let's rebase this to merge into master. @tippenein Thanks!

**Review by kantai — COMMENTED** [review]
LGTM, just rebase to `master` so that it's not pulling in the `develop` changes as well.

**Review by obycode — APPROVED** [review]
Thanks!

**Pull request merged**